docs/releasing.md

```diff
@@ -0,0 +1,86 @@
+# Releases
+
+This document describes how the release flow looks like and how to create an `helm-broker` release.
+
+## Release process
+
+Every change in the master branch which passes all checks creates the `latest` tag and triggers the release.
+
+Every tag pushed to the repository triggers a new release.
+
+### Create a release from the master branch
+
+Follow these steps to create a new `helm-broker` release `v0.6.0`:
+
+1. Checkout on master and pull the latest changes:
+
+    ```
+    git checkout master
+    git pull
+    ```
+2. Prepare the release:
+
+    ```
+    make VERSION=v0.6.0 cut-release
+    ```
+Above command do the following things:
+  - changes the images tag in the `charts/helm-broker/values.yaml` to `v0.6.0`
+  - creates a commit with the changes
+  - creates a `v0.6.0` tag
+
+3. Push the release:
+
+    ```
+    git push {remote} v0.6.0
+    ```
+
+It triggers the Prow CI release job for Helm Broker.
+
+    >**NOTE:** If you want to push the tag to the upstream, run the following command:
+    >```
+    >git push upstream v0.6.0
+    >```
```

Comment:
IMO it better looks in one line:
` > **NOTE:** If you want to push the tag to the upstream, run the following command: ```git push upstream v0.6.0``` `